README.md

```diff
@@ -287,6 +287,7 @@ _Note that Kubernetes annotation maps are all of Go type `map[string]string`.  A
 | <a name="app-whitelist"></a>routable application | service | [router.deis.io/whitelist](#app-whitelist) | N/A | Comma-delimited list of addresses permitted to access the application (using IP or CIDR notation).  These may either extend or override the router-wide default whitelist (if defined).  Requests from all other addresses are denied. |
 | <a name="app-connect-timeout"></a>routable application | service | [router.deis.io/connectTimeout](#app-connect-timeout) | `"30s"` | nginx `proxy_connect_timeout` setting expressed in units `ms`, `s`, `m`, `h`, `d`, `w`, `M`, or `y`. |
 | <a name="app-tcp-timeout"></a>routable application | service | [router.deis.io/tcpTimeout](#app-tcp-timeout) | router's `defaultTimeout` | nginx `proxy_send_timeout` and `proxy_read_timeout` settings expressed in units `ms`, `s`, `m`, `h`, `d`, `w`, `M`, or `y`. |
+| <a name="app-maintenance"></a>routable application | service | [router.deis.io/maintenance](#app-maintenance) | `False` | Whether the app is under maintenance so that all traffic for this app is redirected to a maintenance page if provided. |
```

Comment:
Also:

> Whether the app is under maintenance so that all traffic for this app is redirected to a maintenance page if provided.

After your recent amendments to this PR, that is no longer true... this will _just_ return a 503, without using any maintenance page.